app/templates/support/index.html

```diff
@@ -0,0 +1,52 @@
+{% extends "_base_page.html" %}
+{% block page_title %}What are you trying to do? - Digital Marketplace{% endblock %}
```

Comment:
This question seems a weird title (and heading) for the page - the link to it is "Digital Marketplace support", I think the page title at least could be "Support", and maybe the H1 too, with the question as a lower-level heading.

Reply:
@laurenceberry - can you take a look?

Reply:
Change

What are you trying to do?

TO

What do you need help with?

REASON

It sounds more obviously like support. We could also change the link text on the homepage to DigitalMarketplace help, support is more internal language.

Change

Sell services to

Selling services

Change

Buy services to

Buying services

CHANGE

Create and update

to

Creating and updating 

REASON

Tense needs to change to fit the new question.